src/components/Provider/ServiceTypesCostTable/ServiceTypesCostTable.tsx

```diff
@@ -24,7 +24,9 @@ export const ServiceTypesCostTable = ({ costs, find, recruit, followUp, classNam
 
   return (
     <table className={clsx('govuk-table govuk-!-font-size-16 table-fixed', className)}>
-      <caption className="govuk-table__caption govuk-body-m mb-2">Services available and costs:</caption>
+      <caption className="govuk-table__caption govuk-body-m mb-2">
+        Find, Recruit, Follow-up services available, and costs:
+      </caption>
```

Comment:
the requirement is to make the captions unique and more descriptive but you are only making them more descriptive here. 
to make them unique I would suggest that we bring in the providers name
e.g. add an aria label to the original caption
aria-label="Genomic Profile Register services available and costs"
Since the table is re-useable we will need to find a way to programmatically insert the providers name
e.g.
aria-label=`${providersName} services available and costs`.

I'll send you more info on this